heat/core/linalg/tests/test_svdtools.py

```diff
@@ -4,6 +4,7 @@
 import heat as ht
 import numpy as np
 from mpi4py import MPI
+import time
```

Comment:
Probably can be deleted, guessing it was used during development

Reply:
done :+1: